lib/errors/kuzzleError.js

```diff
@@ -28,7 +32,9 @@ class KuzzleError extends Error {
     return {
       message: this.message,
       status: this.status,
-      stack: this.stack
+      stack: this.stack,
+      code: this.code,
+      errorName: this.errorName
```

Comment:
```suggestion
      name: this.name
```

Reply:
I cannot name it `name` since there is a getter named like that. That's why i choose `errorName`